src/Shouldly.Tests/Shouldly.Tests.csproj

```diff
@@ -5,7 +5,6 @@
     <LangVersion>latest</LangVersion>
     <Optimize>false</Optimize>
     <DebugType Condition=" '$(OS)' == 'Windows_NT' ">full</DebugType>
-    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">2.0.9</RuntimeFrameworkVersion>
```

Comment:
This had to be removed because .NET Core 2.0 was not found on the Ubuntu image. (It is EOL.)